### PR TITLE
EqPly: added --immersive option to use in VR mode

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: examples/eqPly/config.cpp [hunk 1]
```diff
@@ -90,6 +90,9 @@ bool Config::init()
     _loadModels();
     _registerModels();
 
+    if (_initData.centerCamera())
+        _frameData.setCameraPosition(eq::Vector3f::zero());
```
Comment (hernando) [inline examples/eqPly/config.cpp]: Isn't the default camera position (0, 0, 0)? Shouldn't this logic go to wherever the camera is set to something different?

File: examples/triply/vertexBufferRoot.cpp [hunk 2]
```diff
@@ -258,7 +257,8 @@ bool VertexBufferRoot::_constructFromPly(const std::string& filename)
     ++progress;
 
     data.calculateNormals();
-    data.scale(2.0f);
+    if (_rescale)
+        data.scale(2.0f);
```
Comment (hernando) [inline examples/triply/vertexBufferRoot.cpp]: Did you figure out what's the purpose of this scaling? It looks completely arbitrary

Comment (rdumusc) [reply examples/triply/vertexBufferRoot.cpp]: Yes, it is not exactly a scaling but a normalisation to the given size. This works together with the default camera position at z=-2.0 to have a good default view of the model.

File: examples/eqPly/config.cpp (hunk 1 above)
Comment (rdumusc) [reply examples/eqPly/config.cpp]: It isn't, it's z=-2.0, that's what you get when you press the space bar (it goes to (0,0,0) if you press it a second time). There was also a default rotation which I removed in the second commit. I would be possible to invert the logic a bit, but we still need to go to z=-2.0 by default.

**Comment by rdumusc** [comment]
retest this please

**Pull request merged**